addons/website/views/website_templates.xml

```diff
@@ -271,6 +271,28 @@
     </xpath>
 </template>
 
+<template id="cookies_bar" inherit_id="website.footer_custom" name="Cookies Bar">
+    <xpath expr="//div[@id='footer']" position="before">
+        <div id="website_cookies_bar" t-if="website.cookies_bar" class="s_popup o_snippet_invisible" data-name="Cookies Bar">
+            <div class='s_popup_main s_popup_fixed popup_backdrop d-none o_cookies_discrete'
+                 data-show-after="500" data-display="afterDelay" data-consents-delay="999">
```

Comment:
@JKE-be Now I remember about the Python/JS Qweb template sharing, this template is a copy paste of one of the preview template we have in JS Qweb. Not really a problem but a bit of copy/pasted code to maintain